### DISCUSSION TITLE
Running it in docker a container

### DISCUSSION THREAD

**Discussion by jaandk** [svartalf/unbound-telemetry#128] [Q&A]
I have my unbound running in a docker container, and whanted to run the telemetry in a container besides it, but I can't seem to figure out how you would do that, or if it is even posible?

Any help you could give or documentation you could point to would be appreciated, thanks.

**Comment by svartalf** [comment]
Hi, @jaandk !

There is a [`Dockerfile`](https://github.com/svartalf/unbound-telemetry/blob/master/Dockerfile) you can use to create a container with application it.

For [TCP communication](https://github.com/svartalf/unbound-telemetry#tcp-socket) you will also need to mount control keys and certificates into running container, so you will be able to use them, you can check `unbound-telemetry tcp --help` for more information.
Same thing applies to UNIX socket option, but I'm not sure if shared memory option would even work at all (and also it's not something you usually do with containers setup too)

**Reply by jan-di** [reply]
Hi @svartalf,

the Dockerfile looks good, but i would like to have a already built image to use. Do you are open to a PR that adds a github actions pipeline that automatically builds the image and push it to github container registry?
